pytest/tests/sanity/epoch_switches.py

```diff
@@ -16,67 +18,38 @@
 TIMEOUT = HEIGHT_GOAL * 3
 
 config = None
+config_overrides = {
+    "tracked_shards": [0],
+    "view_client_throttle_period": {
+        "secs": 0,
+        "nanos": 0
+    },
+    "state_sync_enabled": True,
+    "store.state_snapshot_enabled": True,
+    "consensus": {
+        "state_sync_timeout": {
+            "secs": 0,
+            "nanos": 500000000
+        }
+    }
+}
+
 nodes = start_cluster(
     2, 2, 1, config,
     [["epoch_length", EPOCH_LENGTH], ["block_producer_kickout_threshold", 40]],
     {
         0: {
```

Comment:
nit: Can you just do `0: config_overrides`? or maybe with .clone() if you're planning on making changes to only one of those.